apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -259,7 +261,7 @@ export async function generateQuestion({
     
     You should provide the following input methods for students to answer: ${promptUserInput}
     
-    To calculate the right answer, you should: ${promptGrading}`;
+    To calculate the right answer, you should: ${promptGrading}. Do NOT display the method to calculate the correct answer in your \`question.html\` unless otherwise requested.`;
```

Comment:
This prompt gets displayed back to the user in the "chat" history. I don't think we want this to be visible to the instructor. Should we move this into the system prompt instead?

Reply:
Moved to system prompt in 080bc0653265594e3a1c00340ffe3cc8f0ed6b1b.